OrbitVulkanLayer/QueueManagerTest.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "QueueManager.h"
+#include "gtest/gtest.h"
+
+#define UNUSED(x) (void)(x)
```

Comment:
Not necessarily a fan of having the macro, I think that using `(void)` directly is a known pattern. You can probably (I think?) also skip the declaration of the variable in the case (if you want), like `(void)manager.GetDeviceOfQueue(queue)`.

Reply:
Ah nice, thanks!